python/py-asteval/Portfile

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem              1.0
+PortGroup               python 1.0
+
+name                    py-asteval
+version                 0.9.12
+categories-append       math
+license                 BSD
+maintainers             @reneeotten openmaintainer
```

Comment:
As of recently, you should be including both your github handle and your obfuscated email address in your maintainer line, as in:

`maintainers    {@reneeotten  domain.com:user} openmaintainer`

Reply:
@pmetzger thanks, I have added an e-mail address.